src/main/java/io/supertokens/inmemorydb/Start.java

```diff
@@ -2687,7 +2686,7 @@ public void insertUsedCode_Transaction(TransactionConnection con, TenantIdentifi
                     Config.getConfig(this).getTotpUsersTable(),
                     new String[]{"app_id", "user_id"},
```

Comment:
see my comment in plugin interface PR

Reply:
I believe you're talking about replacing `TotpNotEnabledError` with new `UnknownUserIdError`. If yes, done.

Reply:
Please check and confirm.

Reply:
Done